pyproject.toml

```
@@ -37,10 +35,12 @@ Sphinx = "^5.0.1"
 sphinx-rtd-theme = "^1.0.0"
 toml = "^0.10.0"
 pytest-asyncio = "^0.23.6"
+pytest-testmon = "^2.1.1"
+pytest-cov = "^5.0.0"
 
 
 [tool.pytest.ini_options]
-addopts = "--doctest-modules"
+addopts = "--doctest-modules --testmon"
```

Actually, it might not always be a good idea to turn this on. I'll revert this change, and devs can turn it on themselves with the `--testmon` flag.
```suggestion
addopts = "--doctest-modules"
```